core-bundle/src/Resources/contao/dca/tl_page.php

```diff
@@ -1154,6 +1154,11 @@ public function generateAlias($varValue, DataContainer $dc)
 			}
 		}
 
+		if (Input::post('VERSION_NUMBER') > 0 && $varValue !== $dc->activeRecord->alias)
```

Comment:
```suggestion
		if ($varValue != $dc->activeRecord->alias)
```

I don't think the version check is required.

Reply:
Ok, the check is removed now.